src/main/webapp/availability.css

```diff
@@ -0,0 +1,24 @@
+#content {
+  margin-left: auto;
+  margin-right: auto;
+  width: 650px;
+}
+
+.form-row {
+  width: 100%;
+  margin-left: 0;
+  margin-right: 0;
+}
+
+.center-with-form {
+  margin-top: 5px;
+  margin-bottom: 0;
+}
+
+.end-time {
+  margin-left: 8px;
+}
+
+.space-above {
```

Comment:
Regarding `.center-with-form`, `.end-time`, and `.space-above` : l don't know if we should name classes based on what they do or what they are applied to. l personally like the latter method, but am unsure.
Thoughts @jalexanderqed @acmcarther ?

Reply:
describing the class by what it does is strictly worse than describing it by what it is.

What it does: "space-above"
What it is: "schedule-card" (just made that up)

Reply:
Done